jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -45,3 +45,23 @@
         branch: master
         job-name: ci-kubernetes-e2e-gce-flaky
         repo-name: k8s.io/kubernetes
+    - kubernetes-e2e-gke-autoscaling:
+        branch: master
+        job-name: ci-kubernetes-e2e-gke-autoscaling
+        repo-name: k8s.io/kubernetes
```

Comment:
Oh.. The way I'm doing it is to keep email, test-owners, and desciption -- might be useful in the future?

And shall we put timeout here rather than in the script which is hard to find?